### ISSUE TITLE
New = Default Kit

### ISSUE THREAD

**Issue by NoAliby** [hydrogen-music/hydrogen#1579]

**Comment by theGreatWhiteShark** [comment]
Hey @NoAliby ,

This is one of the intentional UX changes for the upcoming release. Some users experienced problems with loss of data/modification because of which we reworked the autosave capabilities of Hydrogen a bit and allowed it to automatically recover from unsaved changes applied to a file. When opening a new song / `DefaultSong.h2song` as before and working on it for hours followed by an unexpected electricity outage, the latest changes will be stored in `.DefaultSong.h2song.autosave`. But this recovery can very easily be lost. It['s enough to open a new song, doing a single action, and the first autosave will overwrite the previous progress without the user having time to release it. Therefore, a new song is now a transient entity always bound to a filename which is ensured to not already exist on disk. What you are seeing is the default empty song. Loads of effort to prevent any loss of data :D 

AFAIU your previous routine for creating a new song is to open a template song you created at some point. Would you get the same results when opening the song via the "Open" menu? Sorry for the inconvenience

**Comment by NoAliby** [comment]
Thanks, it will work if I will use a saved file as a template to get the same thing. (I will set the template file to read-only in the file system so that I don't overwrite it.)

_A "Template" feature might be useful in the future - where you can add files that automatically open a file and sets the new project to no-name unsaved so that it will prompt you for a name when you save._

**Comment by theGreatWhiteShark** [comment]
> A "Template" feature might be useful in the future - where you can add files that automatically open a file and sets the new project to no-name unsaved so that it will prompt you for a name when you save.

Yeah, I see your point. How about we convert this issue (by changing the title) into a feature request for the template feature?

**Comment by trebmuh** [comment]
> A "Template" feature might be useful in the future - where you can add files that automatically open a file and sets the new project to no-name unsaved so that it will prompt you for a name when you save.

:+1: 

> Yeah, I see your point. How about we convert this issue (by changing the title) into a feature request for the template feature?

:+1:
